pkg/apis/meta/v1/unstructured/status.go

```diff
@@ -0,0 +1,42 @@
+package unstructured
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+//GetCondition returns the condition with type typeString
+// returns error if the condition is not found
+//u: The *unstructured.Unstructured object to search in
+//typeString: the type to search
+func GetCondition(
```

Comment:
it only get condition by filtering with types? feel the name can be more specific?

Reply:
changing to GetConditionByType

Reply:
Done